packages/client/ts/controllers/info.ts

```diff
@@ -21,18 +21,18 @@ export default class InfoController {
         this.destroyQueue = new Queue();
     }
 
-    create(type: number, data: any[], x: number, y: number): void {
+    create(type: number, data: unknown[], x: number, y: number): void {
         switch (type) {
             case Modules.Hits.Damage:
             case Modules.Hits.Stun:
             case Modules.Hits.Critical: {
-                let damage = data.shift();
-                const isTarget: boolean = data.shift();
-                const dId = this.generateId(this.game.time, damage, x, y);
+                const damage = data.shift() as number | string;
+                const isTarget = data.shift() as boolean;
```

Comment:
```suggestion
                const isTarget = !!data.shift();
```

`isTarget` type will be inferred then.